cmd/query/app/server.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2020 The Jaeger Authors.
```

Comment:
It's not necessary to change the copyright year, but if you do, the correct form would be `2019,2020`

Reply:
fixed